terra-jupyter-base/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # adapted from https://hub.docker.com/r/jupyter/base-notebook/ AKA https://github.com/jupyter/docker-stacks/tree/master/base-notebook
 
-FROM gcr.io/deeplearning-platform-release/tf-gpu.2-7
+FROM gcr.io/deeplearning-platform-release/tf-gpu.2-9
```

Comment:
The updated image contains Conda 4.12.0.

The old image contained Conda 4.11.0 which was causing issues.